src/layouts/CoreLayout/CoreLayout.scss

```diff
@@ -14,9 +14,6 @@
     background-color: $colorSecondary;
   }
 
-  #NavBar {
-  }
-

```

Comment:
This is a preference as well, but personally I prefer having the empty classes in the css. The bundler can remove them, and they make it easier to see what code is styled and what isn't. I also ALWAYS give HTML elements an `id` or `class`, since it'll help other developers figure out the purpose of the element too.